src/screens/AccountBackup.js

```diff
@@ -110,10 +120,15 @@ class AccountBackupView extends React.PureComponent {
           }}
         >
           <Text style={styles.seedText}>
-            {selected.seed}
+            {seedPhrase || seed}
           </Text>
         </TouchableItem>
-        {(isNew) &&
+        {!!derivationPath &&
+          <Text style={styles.derivationText}>
+            {derivationPath}
+          </Text>}
```

Comment:
Not sure that this derivatio password verify should be here or not

<img width="366" alt="Screenshot 2019-08-20 at 16 31 05" src="https://user-images.githubusercontent.com/10432070/63356400-fcb0be00-c367-11e9-9725-9c5aa925d3a8.png">

Reply:
we should not show the password in the UI, but we need to show a hint and a way for users to verify it. I thought about another screen for that, but ended up doing it on the same to avoid more navigation.

Reply:
let's log this as an issue for another PR then